spec/active_force/active_query_spec.rb

```diff
@@ -147,7 +147,7 @@
     let(:quote_input){ "' OR Id!=NULL OR Id='" }
     let(:backslash_input){ "\\" }
     let(:number_input){ 123 }
-    let(:expected_query){ "SELECT Id FROM table_name WHERE (Backslash_Field__c = '\\\\' AND NumberField = 123 AND QuoteField = ''' OR Id!=NULL OR Id=''')" }
+    let(:expected_query){ "SELECT Id FROM table_name WHERE (Backslash_Field__c = '\\\\' AND NumberField = 123 AND QuoteField = '\\\' OR Id!=NULL OR Id=\\'')" }
```

Comment:
This and line 164 should still give the same result if you use only 2 backslashes instead of 3

Reply:
indeed it does - i'll push an update

Reply:
I just pushed to main - https://github.com/openstax/active_force/commit/3210375b8ea95b576c876de87f1e02fcdc0139d0